config/locales/en.yml

```diff
@@ -22,6 +22,7 @@ en:
       files: Files
       new_file: New File
       reset_downloads: Reset Digital Downloads
+      resource_url: Resource URL
```

Comment:
Maybe we should call this "Specify URL manually" so that people understand that it is an *alternative* to adding a file from disk.